Makefile

```diff
@@ -38,6 +38,5 @@ push: image
 	@echo "pushing image"
 	$(RUNTIME) push quay.io/$(REPOOWNER)/$(IMAGENAME):$(IMAGETAG)
 
-.PHONY: test
-test:
-	go test -cover -v ./...
+test-e2e-knit: binaries
```

Comment:
Did you left this target without PHONY on purpose?

Reply:
nope, I missed it. Good catch.